src/main/resources/default/assets/common/scripts/slider.js.pasta

```diff
@@ -248,6 +258,16 @@
         };
 
         const fixView = function () {
+            if (window.innerWidth) {
+                if (window.innerWidth < 992) {
+                    columns = options.columnsXs;
+                } else if (window.innerWidth < 1200) {
```

Comment:
**Question:** Are these magic numbers or are they trying to mirror the bootstrap breakpoints? If its the later they dont match the naming of the sirius or bootstrap breakpoints at all. Can we at least re-use one of the naming schemes to not have a third? 😅

https://getbootstrap.com/docs/5.0/layout/breakpoints/
https://github.com/scireum/sirius-web/blob/e7b0112d7c90ff719f381afcefef1af3c3364320/src/main/resources/assets/design-system/design-system.scss#L1

Reply:
That is the scheme we use within Stories. Of course, we can also use another one.